redesign.md

```diff
@@ -0,0 +1,3 @@
+* Medium-sized buttons use 12px of padding instead of 16px
```

Comment:
Please change the filename to `REDESIGN.md` to match the other files in this directory. Also, can you organize these bullets into the following categories (with headers):

1. Assumptions made
2. Fixes needed
3. Other notes